infra/modules/providers/azure/app-gateway/variables.tf

```diff
@@ -48,15 +69,10 @@ variable "appgateway_ipconfig_name" {
   default     = "subnet"
 }
 
-variable "appgateway_frontend_port_name" {
-  description = "The Frontend Port Name for the Appication Gateway to be created"
-  type        = "string"
-}
-
 variable "frontend_http_port" {
   description = "The frontend port for the Appication Gateway to be created"
   type        = "string"
```

Comment:
can we add a todo to change to a number when we move to terraform 0.12?

Reply:
Good Call 👍